runtests

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 set -e
 cd openwisp-config/tests/
-luacheck . -d -a -r
 lua test_store_unmanaged.lua -v
 lua test_restore_unmanaged.lua -v
 lua test_uci_autoname.lua -v
 lua test_utils.lua -v
 lua test_remove_default_wifi.lua -v
+cd ..
+luacheck . -a
```

Comment:
please ensure the style checks are run before the tests, because we want contributors to know as soon as possible if something is wrong with minor issues, so they can fix quickly

Reply:
@nemesisdesign I did it this way round because if there was a luacheck fail, it wouldn't run any tests and I was unsure if this was wanted or not.
I'll do it the other way round now